usagov/usagov-2021
# Usagov 658 focus color feedback

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-658

## Description

Change the focus color. For the focus, we are going with two strokes. The inner stroke is 2px #FFFFFF. The outer stroke is 2px #0186a2.
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Goto http://localhost/ and tab through the page and verify the focus color is focus is blue #0186a2 with a box-shadow #fff
- The only elements that will not have a box-shadow #fff is the Life Events link “every stage of life” on the homepage and “Most Popular” links navigation pages like http://localhost/benefits
- Also verify the focus on any of these pages below in mobile and Spanish
- Test Pages to verify the focus color
    http://localhost/agency-index
    
    http://localhost/es/ramas-gobierno-estados-unidos

    http://localhost/es/acerca-de-estados-unidos

    http://localhost/state-governments
    
    http://localhost/where-report-scams
    
    http://localhost/where-report-scams/what-type-scam-do-you-need-report#block-usagov-content
    
    http://localhost/elected-officials
    http://localhost/elected-officials-results?input-street=11715+marston+moor+lane&input-city=waldorf&input-state=MD&input-zip=20602#skip-to-h1
    
    http://localhost/elected-officials-email?input-street=11715+marston+moor+lane&input-city=waldorf&input-state=MD&input-zip=20602&email=correspondence%40vanhollen.senate.gov&name=Chris+Van+Hollen&office=U.S.+Senator#skip-to-h1

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
